clients/da-vinci-client/src/main/java/com/linkedin/davinci/blobtransfer/NettyP2PBlobTransferManager.java

```diff
@@ -26,147 +25,179 @@
  */
 public class NettyP2PBlobTransferManager implements P2PBlobTransferManager<Void> {
   private static final Logger LOGGER = LogManager.getLogger(NettyP2PBlobTransferManager.class);
-  protected static final int MAX_RETRIES_FOR_BLOB_TRANSFER_PER_HOST = 3;
-  protected static final int MAX_TIMEOUT_FOR_BLOB_TRANSFER_IN_MIN = 60;
+  // Log messages format
+  private static final String NO_PEERS_FOUND_ERROR_MSG_FORMAT =
```

Comment:
nit: Thank you for adding the log pattern formatting - since you already try to format the logging, could we use Utils.getReplicaId() API to log all replica logs? This will help us to build log dashboard to filter all ingestion related logs for a certain replica, when we want to have a clear timeline to analyze.

Reply:
+1

Reply:
IMO having too many format specifiers could be confusing and leads to wrong order of types which can cause prod issues. For log4j reporting using `{}` is a safer choice. Only for exception messages we might need them, which again can be skipped using string concatenation.

Reply:
Thank you @sixpluszero @adamxchen @majisourav99 for the suggestion. PR is updated with using `Utils.getReplicaId()` in log and using `{}` instead.